article/find_nearest_document.py

```diff
@@ -0,0 +1,22 @@
+from operator import itemgetter
+import numpy as np
+
+NEAREAST_DOCS_AMOUNT = 5
+
+def get_neareast_docs(input):
+    i = 0
+    similarity_array = []
+    for j in input:
```

Comment:
Jeg vil nok bruge noget andet end `k` fx: `main_doc`. `k,i,j` bruger man kun til traditionel for loops hvor man bruger `[]` eller i matematik.